pkg/didcomm/packer/jwe/authcrypt/pack.go

```diff
@@ -252,7 +253,8 @@ func (p *Packer) encodeRecipient(cek, recipientPubKey, senderPubKey *[chacha.Key
 }
 
 // buildRecipient will build a proper JSON formatted and JWE compliant Recipient
-func (p *Packer) buildRecipient(key string, apu []byte, spkEncoded, nonceEncoded, tagEncoded string, recipientKey *[chacha.KeySize]byte) (*jose.Recipient, error) { //nolint:lll
+func (p *Packer) buildRecipient(key string, apu []byte, spkEncoded, nonceEncoded, tagEncoded string,
+	recipientKey *[chacha.KeySize]byte) (*jose.Recipient, error) {
```

Comment:
`chacha.KeySize` ?

Reply:
this is old code, I just removed the nolint directive

Reply:
regardless - it is confusing for what is being implemented... old code should be adjusted to fit with changes too :).

Reply:
Reverted this change.